src/utils/uiUtils.ts

```diff
@@ -64,4 +77,39 @@ export class UIUtils {
             quickPick.show();
         });
     }
+
+    static getWorkspaceDir(): string {
+        const workspaceFolders = workspace.workspaceFolders;
+        if (!workspaceFolders || workspaceFolders.length === 0) {
+            throw new NoWorkspaceError(
+                'No workspace defined for this project.'
+            );
+        }
+        return workspaceFolders[0].uri.fsPath;
+    }
+
+    static async getStaticResourcesDir(): Promise<string> {
+        return new Promise<string>(async (resolve, reject) => {
+            let projectPath: string;
+            try {
+                projectPath = this.getWorkspaceDir();
+            } catch (err) {
+                return reject(err);
+            }
+            const staticResourcesPath = path.join(
+                projectPath,
+                this.STATIC_RESOURCES_PATH
+            );
+            try {
+                await access(staticResourcesPath);
+            } catch (err) {
+                const accessErrorObj = err as Error;
+                const noAccessError = new NoStaticResourcesDirError(
+                    `Could not read landing page directory at '${staticResourcesPath}': ${accessErrorObj.message}`
```

Comment:
Nit: we should probably change this error message to something more generic, now that it's split out into a utility function. E.g.:

> Could not read static resources directory at '${staticResourcesPath}'